CHANGELOG.md

```diff
@@ -1,5 +1,182 @@
 # Changelog
 
+## 1.0.2 
+
+### Additions and Improvements
+
+- Removed DB init when using `public-key` subcommand [\#1049](https://github.com/PegaSysEng/pantheon/pull/1049) 
+- Added `public-key address export` subcommand [\#888](https://github.com/PegaSysEng/pantheon/pull/888) 
```

Comment:
#888 was already in 1.0.1, that's the one we added and was the reason for the 1.0.1

Reply:
Fixed